src/main/frontend/src/assets/locales/en.json

```
@@ -427,29 +427,29 @@
       },
       "keywords": {
         "_": "Keywords",
-        "info": "These are special words that will return your project when people add them to their searches. Max 5."
+        "info": "These are special words that will return your project when people add them to their searches. (max 5 keywords)"
       },
       "source": {
         "_": "Source code",
-        "info": "Support the community of developers by making your project open source!"
+        "info": "A link to your project's source code. Support the community of developers by making your project open source!"
       },
       "_": "Settings",
       "deployKey": {
         "_": "Deployment key",
-        "info": "Generate a unique deployment key to enable build deployment from Gradle"
+        "info": "Generate a unique deployment key to enable build deployment from Gradle."
       },
       "description": {
         "_": "Description",
-        "info": "A short description of your project (max 120)"
+        "info": "A short description of your project. (max 120 characters)"
       },
       "forumSync": {
         "_": "Create posts on the forums",
-        "info": "Sets if events like a new release should automatically create a post on the forums"
+        "info": "Should events (like new releases) have a forum post automatically created?"
```

```suggestion
        "info": "Should events like new releases automatically create a forum post?"
```
@KennyTV Slightly different and reads better to me, not sure what others think though